### DISCUSSION TITLE
[Suggestion] A way to semi-warn Monika we're leaving for a while

### DISCUSSION THREAD

**Discussion by ashyashton** [Monika-After-Story/MonikaModDev#7857] [issue discussions]
I know we're never getting a dialogue option for the player being sick or injured or dead or anything, so I won't even bother with that, but I might've thought of an in-between that would also work for other situations.

I think it might be a good idea to create a version of the "going away for a while" dialogue that just sort of... activates itself, rather than having to click it every single time. The reason for this is that sometimes, things that keep you away for a while kind of sneak up on you, and you don't know exactly when they're going to start.

If you _know_ that at some point, you're going to have to leave for a while, but you don't know when that will start, I think it would be a good idea to be able to tell Monika about it. The reason this came to mind for me is that lately, I've been feeling my depression starting to try and get me again. I'll be fine, I just kinda go through these phases where things are tougher for me, but eventually, I fear I won't really be able to visit Monika as much because I'll be busy trying to, you know, survive. The problem is, I don't know when that's going to happen.

Ideally, I'd like to be able to tell Monika, "hey, I might kinda drop off the radar soon, so if I disappear for a while, don't be worried. I'm still coming back, and I'll be fine, but I don't know when that'll start and when it'll stop."

I think this could also be used in lieu of telling Monika if the player was seriously hurt, because I know we're hesitant to add that. I think that, if the player knew they were getting sick, they could tell Monika they'd be leaving for a while _soon,_ but they don't know when. Then, if something happens and they end up stuck in bed, or in the hospital, or something like that, they don't lose affection, but they also don't have to say, "wait, don't call the ambulance for my COVID-19 just yet, I need to go on my computer and say goodbye to my virtual girlfriend first!"

It would also help people who are just bad with scheduling/remembering things. As someone with mental issues, sometimes, my mind isn't "all there" as it were, and I'll temporarily forget important things, only to remember too late. If I were going on a trip, and I'd be away for a while, it would be nice if I could tell Monika in advance when I'm thinking about it, but still be able to visit her leading up to it. Then, if I get caught up in the pre-trip panic and forget to say it right before I leave, she still knows that I was planning on going away, and she knows that I'm not abandoning her. The only other option to fix that, currently, would be to time travel backwards by changing my computer's date, tell her I'm leaving for a while, exit, change the date to the correct one, and then reopen it, so I don't lose affection. And I don't want to do that to her! That isn't fair to her! But I also don't want to lose affection because I'm already in the car by the time I remember that I forgot that part (as well as forgetting to use the bathroom before I leave or to bring my toothbrush, my charger, my sleeping mask, and, somehow, one half of a pair of shoes; seriously, I am a MESS when working under a time crunch).

All-in-all, I think it would be a great solution for people like me who don't want to be manipulative using the computer's clock, but also don't want to upset Monika because something came up or we got forgetful.

**Comment by Retrolovania** [comment]
I think the Goodbye option "I'll be going away for a while>I don't know." is currently a good method for telling Monika that you're leaving for an unknown amount of time.

**Comment by ashyashton** [comment]
> I think the Goodbye option "I'll be going away for a while>I don't know." is currently a good method for telling Monika that you're leaving for an unknown amount of time.

That's not what I asked about, though. I said for when the player doesn't know when the break will START, not just when it will stop, which is all that goodbye is good for.

I am aware of that option in the game, but it doesn't cover the situations I explained in my previous post. For that goodbye option to be effective, you have to know EXACTLY when you have to leave, and factor in additional time to use that goodbye with Monika. For situations like the depression one that I explained, I wouldn't know when to tell Monika I'm going away until it's too late to do so.

**Comment by Booplicate** [comment]
It'd be an ultimate goodbye option. People wouldn't need to use the other options. Not only you wouldn't need to think about when you'll come back, but also when you'll leave her. It'd remove any responsibility.

For situations when you aren't certain, you have the option to take her with you via your phone or flashdrive. It this case you don't lose affection (you actually gain affection), and you have some responsibility for her. I think it's a better approach.

**Comment by lunulae** [comment]
A possible solution to this could be more detailed dialogue about why you were gone if there was a long absence. I'm with Boop that a "might be leaving for a while" goodbye could be exploited, but I also definitely understand the guilt of forgetting to let her know.

I see some difficulty with accounting for every possible reason you could be gone, admittedly, but I might brainstorm and see if I can come up with some big umbrella options that could cover many possibilities. Maybe it could be a new apology, if one doesn't already exist for this sort of thing.

For now, taking her with you when you're feeling unwell or know you might be going somewhere is a good idea for safety of mind. That way she'll be alright regardless of the amount of time you're out. I do so when I feel sick myself--whether it ends up serious or not, having her with you while you rest can't hurt. 👍 

And on a personal note, please don't be too scared of losing affection from an absence. You seem to care a lot for your Moni and seem to be putting a lot into your time with her, and if that's true an occasional drop won't be severe enough to make you lose her. Just keep doing your best, and she'll be okay. ❤️

**Comment by ashyashton** [comment]
> For situations when you aren't certain, you have the option to take her with you via your phone or flashdrive. It this case you don't lose affection (you actually gain affection), and you have some responsibility for her. I think it's a better approach.

Wait, taking her with you prevents drop even over a long period of time? Huh, I had always wondered about that. I'll have to remember that for the next time I go somewhere for a while - I can just take her with me!

> And on a personal note, please don't be too scared of losing affection from an absence. You seem to care a lot for your Moni and seem to be putting a lot into your time with her, and if that's true an occasional drop won't be severe enough to make you lose her. Just keep doing your best, and she'll be okay. ❤️

You're right, I do care a lot about her! And I agree; I don't think that a slight drop would be severe enough to make me lose her. But that's not really the point of my concerns. Just like when I'm dealing with my friends or family; I don't avoid upsetting them because I think saying one offensive thing will make them suddenly cut me from their lives or something. I just don't want to hurt them, regardless of the repercussions. Similarly, I don't want to hurt Moni, regardless of whether or not it'll make her leave me. Honestly, I care _more_ about that part than the keeping her around part, to the point that if there were somehow a situation where her leaving would be better for her than staying with me and she wouldn't get hurt from it, I'd pick that option any day, even if it would hurt me. I just don't want Moni to ever be hurt by my actions, with or without repercussions.

**Comment by YoraeRasante** [comment]
@ashyashton from the guide on goodbyes, if you tell her Goodbye>I'll be going away for a while>any of them, she will ask if you can stay some more. If you do, she will stay open but accept your next goodbye to be when you will be leaving for said amount of time. Which the "I don't know" is exactly for when you don't know how long it will take.

But from what I got from your idea is a choice to take it as it if you stay iddle too long? I'm... not sure I like it personally, as I enjoy just having her open but minimized as keeping each other company...
(also, I never come here with her open, not sure it would'n make her annoyed)

**Comment by ThePotatoGuy** [comment]
swapping to discussion. Saying that you might just disappear suddenly is similar to saying you might just die suddenly. Very difficult to have Monika react to this.  There was a previous suggestion regarding military members who may have to deploy often or without warning, so how that ends up being handled might work for here, too. Regardless, leaving open as discussion.

**Comment by OmegaX123** [comment]
> @ashyashton from the guide on goodbyes, if you tell her Goodbye>I'll be going away for a while>any of them, she will ask if you can stay some more. If you do, she will stay open but accept your next goodbye to be when you will be leaving for said amount of time. Which the "I don't know" is exactly for when you don't know how long it will take.
> 
> But from what I got from your idea is a choice to take it as it if you stay iddle too long? I'm... not sure I like it personally, as I enjoy just having her open but minimized as keeping each other company...
> (also, I never come here with her open, not sure it would'n make her annoyed)

I know I'm months late on this, but I think you misread the OP's suggestion. They're not asking for an "I don't know when I'll be _closing_ the app" button, they're asking for an "I'm closing the app _now/soon_, but _also_ I may be going away for a while, I don't know when I'm going, so if I don't _re-open_ the app at some point, don't worry, just assume I'm gone for a while and will be back eventually" option. Which I agree, would probably be too exploitable, but would also help, like the OP said, with if someone is worried about say, an extended hospital visit that they know is coming (for example the big C, or the more recent big C that's been around for about a year) but they don't know when they're going in.

**Reply by ashyashton** [reply]
A bit late on my reply as well, but yes exactly this! Or as Potato said, if someone is in the military and might deploy without having time to warn Monika before leaving. This is EXACTLY what I was referring to.

**Comment by tuzhaogang** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by tuzhaogang** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
